profiles/models.py

```diff
@@ -1,3 +1,16 @@
 from django.db import models
 
-# Create your models here.
+from users.models import Users
+
+
+class StartUpProfile(models.Model):
+    user_id = models.ForeignKey(Users, on_delete=models.CASCADE)  # model User is empty
+    name = models.CharField(unique=True, max_length=255, blank=False)
+    description = models.TextField()
+    website = models.CharField(max_length=255)
+    startup_logo = models.BinaryField()
```

Comment:
Fields like `website` and `startup_logo` may be optional, so you could explicitly set `null=True`, `blank=True` to handle that.